extractor.js

```diff
@@ -72,11 +72,11 @@ class Extractor {
 
     async compressOutput(){
         this.packagedFile = tmp.fileSync();
-        await compressing.tar.compressDir(this.projectOutputDir.name, this.packagedFile.name)
-            .then(result=>result)
+        await compressing.tar.compressDir(this.projectOutputDir.name, this.packagedFile.name, { ignoreBase: true })
             .catch(function(reason){
```

Comment:
eta-conversion: `.catch(console.log)`

Reply:
Do you actually want to catch this? There's no error handling going on.

Reply:
for now, I feel it's appropriate to log errors, there's still value since the plugin is still in its early stages.
No, it doesn't serve any production purpose aside from debugging